etc/config/compiler-explorer.amazon.properties

```diff
@@ -1,4 +1,4 @@
-# Settings overridden for Amazon EC2 instances that power http://*.godbolt.org/
+# Settings overridden for Amazon EC2 instances that power http://*.compiler-explorer.com/
```

Comment:
Would it maybe be better to just say power the live site since we have many domains?